src/pages/Home.jsx

```diff
@@ -8,12 +8,12 @@ function Home () {
       {/* Mobile Version */}
       <div className='lg:hidden'>
         <div className='flex bg-white py-2.5'>
-          <p className='font-normal font-header text-xl ml-auto mx-2'>
-            Tap to call!
-          </p>
-          <p className='font-normal font-header text-xl text-green-700 mx-2'>
-            123.456.7890
+          <p className='mx-2 ml-auto text-xl font-normal font-header'>
+            Tap to email!
           </p>
+          <a href="mailto:info@yorkshirewoodsgroundcrew.com" className='mx-2 text-xl font-normal text-green-700 font-header'>
+            info@yorkshirewoodsgroundcrew.com
+          </a>
```

Comment:
It looks like the link in the mobile version might not be positioned quite right; we should make sure it fits well on smaller screens.

![mobile](https://github.com/cherryontech/ywgc/assets/107000157/2357c185-7d56-4bc9-8093-15a0a87ec2b1)

Reply:
Good catch. I got updated designs from the designer so we can avoid this.